### PR TITLE
fixed security vulnerability issue

### PR DESCRIPTION
**Pull request by steven-xu-lf**
- updated json5 dependency to the latest version in order to fix the security dependency issue in the `package-lock.json` file since I noticed that it only exists in the `package-lock.json` file
- updated project version
- updated changelog

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@laserfiche/lf-api-client-core",
-  "version": "1.1.7",
+  "version": "1.1.8",
```
Comment (jason-chan-lf) [inline package.json:3]: can you also update the pipeline version number

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 1.1.8
+
+### Fixes
+- Fixed security vulnerability issue by updating json5 dependency to version 2.2.3 in the `package-lock.json` file
```
Comment (jason-chan-lf) [inline CHANGELOG.md:4]:
```suggestion
- Fixed security vulnerability issue by updating json5 dependency to version 2.2.3
```
I don't think we need to be specify the file we are fixing. Just mentioning the outcome of the fix is good

File: package.json (hunk 1 above)
Comment (steven-xu-lf) [reply package.json:3]: ok done, thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (steven-xu-lf) [reply CHANGELOG.md]: ok done, thanks!

**Pull request merged**